modules/virt-about-cpu-and-memory-quota-namespace.adoc

```diff
@@ -7,5 +7,5 @@
 
 A _resource quota_, defined by the `ResourceQuota` object, imposes restrictions on a namespace that limit the total amount of compute resources that can be consumed by resources within that namespace.
 
-The `CDI` object defines the user configuration for the Containerized Data Importer (CDI). The CPU and memory request and limit values for the `CDI` object are set to a default value of 0.
+The `HyperConverged` custom resource (CR) defines the user configuration for the Containerized Data Importer (CDI). The CPU and memory request and limit values are set to a default value of 0.
```

Comment:
@aglitke Is this still accurate? Thanks!

Reply:
I think it is superfluous to name individual components that HCO configures.  The point of the feature is to concentrate all configuration in one place regardless of the underlying details.

Reply:
You are probably right that we're giving too much detail there, @aglitke, and I'll take that into consideration. I meant to ask if the statement "The CPU memory request and limit values are set to a default value of 0" is still accurate. Thanks!